pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -106,6 +106,7 @@ discovery.relabel "kubernetes_pods" {
 	{{- if .SupportPodLogs }}
 	rule {
 		source_labels = ["giantswarm_observability_tenant"]
+		regex         = "({{ join "|" .Tenants }})"
```

Comment:
I'm not sure a fond of this regex as it's not stating why we are dropping the logs 🤔

Reply:
Do you mean we should add a comment here ?

Reply:
Oh sorry, I think we should rather use https://grafana.com/docs/alloy/latest/reference/components/loki/loki.process/#stagedrop instead to be able to set the `drop_counter_reason` then it would make it easy to craft an alert around it instead of trying to figure out why the logs are not ingested. It would be better UX wise IMO

Reply:
That would be great, I'll try this.